### PR TITLE
Add an OSC server

### PR DESCRIPTION
**Pull request by thecadams**
Server just outputs to stdout for now, but it supports both Muse Monitor and Muse Direct mobile apps' OSC formats.

### CODE REVIEW

**Comment by thecadams** [comment]
Most of this is a mashup from https://github.com/attwad/python-osc - huge props to @attwad for the great work on the library!

File: oscserver/osc_message.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    @property
+    def address(self):
+        """Returns the OSC address regular expression."""
+        return self._address_regexp
+
+    @staticmethod
+    def dgram_is_message(dgram):
+        """Returns whether this datagram starts as an OSC message."""
+        print("is_message", dgram.startswith(b'/'))
```
Comment (wilfamir) [inline oscserver/osc_message.py]: nitpick - worth using logging to make sure we can control the logging handler?

**Comment by thecadams** [comment]
Addressed your comments @wilfamir @ehborisov feel free to take another look.

File: oscserver/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+python-osc
```
Comment (ehborisov) [inline oscserver/requirements.txt]: maybe a setup.py script instead?

from setuptools import setup

setup_config = {
    'description': 'OSC server handling a stream of data in OSC format.',
    'packages': ['oscsevrer'],
    'install_requires': ['python-osc'],
    'name': 'osc-server'
}


def main():
    setup(**setup_config)


if __name__ == '__main__':
    main()

**Comment by thecadams** [comment]
Addressed comments. Ready to merge!

**Pull request merged**